examples/banananation/src/main/java/gov/nasa/jpl/aerie/banananation/Configuration.java

```diff
@@ -23,6 +29,12 @@ public boolean validateInitialDataPath() {
   }
 
   public static @Template Configuration defaultConfiguration() {
-    return new Configuration(DEFAULT_PLANT_COUNT, DEFAULT_PRODUCER, DEFAULT_DATA_PATH);
+    return new Configuration(DEFAULT_PLANT_COUNT, DEFAULT_PRODUCER, DEFAULT_DATA_PATH, DEFAULT_INITIAL_CONDITIONS);
   }
+
+  @AutoValueMapper.Record
+  public record InitialConditions(LinearResource fruit, double peel, Flag flag) {}
+
+  @AutoValueMapper.Record
+  public record LinearResource(double initial, double rate) {}
```

Comment:
Would it be possible to use the `RealDynamics` class in Merlin Framework? I'd rather not have the modeller make a basic common structure like this if we can avoid it.

Reply:
Absolutely - this was merely meant as an example of a nested record. I can come up with something else to demonstrate the nested record to avoid duplication/confusion with RealDynamics

Reply:
I've removed the nested record - this functionality is covered by `RecordParameter`.